sherpa/models/tests/test_model.py

```diff
@@ -1079,6 +1108,109 @@ def calc(self, p, *args, **kwargs):
         return numpy.ones(args[0].size)
 
 
+def get_cache_classes():
+    """This is a function because we want to conditionally
+    include an XSPEC model. Within a function, we can simply
+    pass that if XSEPC is not available.
```

Comment:
```suggestion
    pass that if XSPEC is not available.
```

The number of times I make this tpyo!